### PR TITLE
Lk pd 2229 update optimus fastqprocessing

### PR DESCRIPTION
**Pull request by ekiernan**
### Description
Updated the Optimus pipeline to include modified inputs for the Fastqprocessing task. Fastq processing now takes in read structure instead of barcode and UMI lengths. 

We added a check to the CheckInputs task so that Optimus will choose the correct read structure based on the 10x chemistry selected as input. 

This work relates to the ticket https://broadworkbench.atlassian.net/jira/software/projects/PD/boards/167?selectedIssue=PD-2229

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 5.8.1
+2023-05-04 (Date of Last Commit)
+
+* Defeated the rebel scum
```
Comment (nikellepetrillo) [inline pipelines/skylab/optimus/Optimus.changelog.md:4]: is this a star wars reference? 😆

Comment (ekiernan) [reply pipelines/skylab/optimus/Optimus.changelog.md:4]: yes! Will revert to professionalism.

**Comment by nikellepetrillo** [comment]
retest this please

**Review by nikellepetrillo — APPROVED** [review]

**Review by aawdeh — APPROVED** [review]
LGTM!

**Pull request merged**